frontend/src/views/Raid.vue

```diff
@@ -15,10 +15,22 @@
                 Total Power
                 <span class="badge badge-primary badge-pill">{{ totalPower }}</span>
               </li>
-               <li class="list-group-item d-flex justify-content-between align-items-center raid-details-text">
+              <li class="list-group-item d-flex justify-content-between align-items-center raid-details-text">
                  Boss Power
                 <span class="badge badge-primary badge-pill">{{ bossPower }}</span>
               </li>
+              <li class="list-group-item d-flex justify-content-between align-items-center raid-details-text progress-container">
+                <div class="d-flex justify-content-between align-items-center raid-details-text" style="width: 100%;">
```

Comment:
Not a big deal, but any reason for not specifying a single class for this
`d-flex justify-content-between align-items-center raid-details-text`?

It's even used twice
```
<li class="list-group-item d-flex justify-content-between align-items-center raid-details-text progress-container">
   <div class="d-flex justify-content-between align-items-center raid-details-text" style="width: 100%;">
```